acquistion_of_technology/common/buildings/z_aot_stellarite_buildings.txt

```diff
@@ -2510,207 +2629,217 @@ building_stellarite_stronghold = {
+            acot_sr_dark_energy = 1000
+        }
+        produces = {
+            acot_sr_stellarite = 25
+        }
+        upkeep = {
+            trigger = {
+                owner = {
+                    is_ai = no
+                }
+            }
+            acot_sr_dark_energy = 50 #Mod Pack; reduced from 100
+            sr_dark_matter = 50 #Mod Pack; reduced from 100
+            energy = 100 #Mod Pack; added
+        }
+    }
+    prerequisites = {
+        "tech_stellarite_nexus"
+    }
+    ai_weight = {
+        weight = 10
+        modifier = {
+            factor = 0
+            owner = {
+                is_country_type = awakened_fallen_empire
+            }
+        }
+        modifier = {
+            factor = 10000
+            owner = {
+                has_monthly_income = {
+                    resource = acot_sr_stellarite
+                    value <= 1
+                }
+            }
+        }
+        modifier = {
+            factor = 0
+            num_buildings = {
+                type = building_stellarite_nexus
+                value >= 2
+            }
+        }
+    }
+    ai_resource_production = {
+        acot_sr_stellarite = 500
+        trigger = {
+            always = yes
+        }
+    }
 }
 
 building_stellarite_generator = {
-	base_buildtime = 600
-	can_demolish = yes
-	category = manufacturing
-	potential = {
-		exists = owner
-		owner = {
-			has_technology = tech_stellarite_reactor
-		}
-	}
-	allow = {
-		has_major_upgraded_capital = yes
-	}
-	triggered_planet_modifier = {
-		potential = {
-			exists = owner
-			owner = {
-				is_gestalt = yes
-			}
-		}
-		modifier = {
-			job_acot_stellaritedrone_add = 2
-		}
-	}
-	triggered_planet_modifier = {
-		potential = {
-			exists = owner
-			owner = {
-				is_gestalt = no
-			}
-		}
-		modifier = {
-			job_acot_stellariteman_add = 2
-		}
-	}
-	resources = {
-		category = planet_buildings
-		cost = {
-			minerals = 1150
-			energy = 1150
-		}
-		produces = {
-			trigger = {
-				owner = {
-					is_ai = yes
-				}
-			}
-			acot_sr_stellarite = 100
-			energy = 200
-		}
-		upkeep = {
-			trigger = {
-				owner = {
-					is_ai = no
-				}
-			}
-			energy = 10
-		}
-	}
-	prerequisites = {
-		"tech_stellarite_reactor"
-	}
-	upgrades = {
-		building_stellarite_generator_ae
-	}
-	ai_weight = {
-		weight = 150000
-		modifier = {
-			factor = 100000
-			owner = {
-				has_monthly_income = {
-					resource = acot_sr_stellarite
-					value <= 3
-				}
-			}
-		}
-		modifier = {
-			factor = 0.1
-			num_buildings = {
-				type = building_stellarite_generator
-				value >= 2
-			}
-		}
-		modifier = {
-			factor = 0
-			owner = {
-				is_country_type = awakened_fallen_empire
-			}
-		}
-	}
+    base_buildtime = 600
+    can_demolish = yes
+    category = manufacturing
+    potential = {
+        exists = owner
+        owner = {
+            has_technology = tech_stellarite_reactor
+            NOT = {
+                has_modifier = resort_colony
```

Comment:
Same deal here with adding the Penal Colony to the exclude list as well. If you want to exclude more buildings from Resort/Penal worlds I'd also suggest moving these triggers into their own scripted trigger in file `z_aot_scripted_triggers_buildings.txt` this will save you copying the same lines over and over. Also should it become necessary in the future to add more buildings to this list you only do it once in the scripted trigger.